homeassistant/components/hassio.py

```diff
@@ -143,6 +192,9 @@ def update_hass_api(self, http_config):
             'password': http_config.get(CONF_API_PASSWORD),
         }
 
+        if CONF_SERVER_HOST in http_config:
+            _LOGGER.warning("Don't use 'server_host' options with Hass.io!")
```

Comment:
This is a warning because it messes with the watch dog right? Can we disable the watch dog temporarily when we see that this config is set?

Reply:
A branch of add-on use the internal HomeAssistant proxy and they will not work if that is enable. So a lot of hass.io things going wrong is that used. Anyway on hass.io it make no sense to use this options.

The warning is, that user know that a lot of things will not work correct. I will disable the API watchdog in this case 👍